main_27_7_2018.sh

```diff
@@ -34,26 +38,50 @@ sample_accessions=("SRR1258218" "SRR1153470")
 ref_vcf_url="ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/NA12878_HG001/latest/GRCh38/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_PGandRTGphasetransfer.vcf.gz"
 ref_genome="ens_build38_release92"
 ref_genome_url='ftp://ftp.ensembl.org/pub/release-92/fasta/homo_sapiens/dna/Homo_sapiens.GRCh38.dna.chromosome.*.fa.gz' ## primaly assembly has unplaced contigs while the vcf has only chromosomes
+ref_annot='ftp://ftp.ensembl.org/pub/release-92/gtf/homo_sapiens/Homo_sapiens.GRCh38.92.gtf.gz'
 
 ## download GIAB data using fastq-dump
 cd $work_dir/data
 module load GNU/4.4.5
 module load SRAToolkit/2.8.2
-for sample_accession in ${sample_accession[@]};do
-  fastq-dump --defline-seq '@$sn[_$rn]/$ri' --split-files $sample_accession
-done
+wget ftp://ftp-trace.ncbi.nih.gov/sra/sra-instant/reads/ByRun/sra/SRR/SRR115/SRR1153470/SRR1153470.sra
+wget ftp://ftp-trace.ncbi.nih.gov/sra/sra-instant/reads/ByRun/sra/SRR/SRR125/SRR1258218/SRR1258218.sra
 wget $ref_vcf_url -O $work_dir/ref_vcf/$sample.vcf.gz
+gunzip $ref_vcf/$sample.vcf.gz#gunzip
```

Comment:
remove the comment (In the general, their should be a space before the hash character)